src/CreatePackage.php

```diff
@@ -0,0 +1,181 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/maintainers for the canonical source repository
+ * @copyright Copyright (c) 2017 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/maintainers/blob/master/LICENSE.md New BSD License
+ */
+
+namespace ZF\Maintainer;
+
+use RecursiveDirectoryIterator;
+use RecursiveIteratorIterator;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\ConsoleOutputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+
+class CreatePackage extends Command
+{
+    protected function configure()
+    {
+        $this
+            ->setName('create-package')
+            ->setDescription('Creates a new package')
+            ->setHelp('here is the help...')
```

Comment:
Either omit this or put something useful in it. :smile:

Reply:
Oh ! Forgot to put details here ! Good catch, thanks!